restricted_site_access.php

```diff
@@ -255,6 +255,9 @@ public static function restrict_access( $wp ) {
 		self::$rsa_options = self::get_options();
 		$is_restricted     = self::is_restricted();
 
+		// Check if is REST API
+		if ( $wp->request && strpos( $wp->request, "wp-json/") === 0 ) return;
```

Comment:
does this check work if pretty permalinks are disabled? rest urls use query parameters in that case and this might fail.